### PR TITLE
Fix for breaking change to gophermap parsing for the upcoming spacecookie release

### PR DESCRIPTION
**Pull request by sternenseemann**
The old parsing behavior was unfortunately not completely in line with the behavior of Bucktooth and pygopherd. Since their format is much more convenient for the user, I have opted to break parsing for some situations in the upcoming release in order to have more correct parsing.

This PR contains a few general fixes related to building and installing and a commit to reflect the changes in spacecookie. Feel free to wait with merging until the release is out if you're not comfortable depending on a git version of spacecookie.

Below is the relevant commit message:

```
For spacecookie >= 0.3.0.0 prefixing with 'i' is no longer needed

The changes are described in detail in the changelog:
https://github.com/sternenseemann/spacecookie/blob/master/CHANGELOG.md#changes-to-networkgopherutilgophermap

In a nutshell we now require a gophermap line which links to another
gopher resource to contain at least one tab character. This makes them
easily distinguishable from info lines and resolves the issue where an
info line would be parsed incorrectly if it started with a valid gopher
file type character.

Unfortunately this also breaks the previous workaround for the issue
which was used in burrow:

  iMy text bla bla

This now can be written as

  My text bla bla

or

  iMy text bla bla\t

to achieve the same result.

This commit adjusts burrow's code for the first variant and updates a
few documentation links.
```

### CODE REVIEW

**Comment by hyperrealgopher** [comment]
Thank you so much for this quality commit! Sorry I took so long to get around to looking it over and merging it! Great work--very much appreciated!
